modules/Kconfig.nordic

```diff
@@ -75,7 +75,6 @@ endmenu
 
 config HAS_NRFX
 	bool
-	select HAS_CMSIS_CORE
```

Comment:
I think this is not only redundant for nRF SoCs but would also be a problem, in case nRFX would need to support SoCs that are not of ARM Cortex-M architecture ;)

Reply:
...just saying @jfischer-phytec-iot :)